### PR TITLE
fix(navbar): alignments

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/Talend/bootstrap-theme/blob/master/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

screen shot 2016-12-16 at 15 32 09

**What is the new behavior?**

screen shot 2016-12-19 at 13 59 06
screen shot 2016-12-19 at 14 00 00

### CODE REVIEW

**Comment by mmaillart** [comment]
The logo and the rest the top bar (Title and such) should also be vertically aligned, as it was before. But with the new size of the logo (bigger)

**Comment by acateland** [comment]
@mmaillart why not providing the logo in svg format ? to avoid bluriness on retina display.

**Comment by mmaillart** [comment]
@acateland : the logo is available in SVG. Up to us to integrate it.

**Comment by mmaillart** [comment]
This looks good to me. Tx !

File: src/theme/_navbar.scss [hunk 1]
```diff
@@ -5,14 +5,15 @@
...
+			font-size: 15px;
 			letter-spacing: 0.7px;
 			text-transform: uppercase;
 
 			&:before {
 				content: '';
 				float: left;
-				height: 100%;
+				height: 125%;
+				margin-top: -4px;
```
Comment (frassinier) [inline src/theme/_navbar.scss]: I'm not a huge fan of this line. Can you explain?

Comment (acateland) [inline src/theme/_navbar.scss]: @frassinier magic number to align an image in such way that text baseline inside that image is aligned to html text inside the bar.

Comment (jmfrancois) [inline src/theme/_navbar.scss]: no logo is ok for you @frassinier ?

Comment (jmfrancois) [inline src/theme/_navbar.scss]: or what solution do you propose ?

Comment (frassinier) [inline src/theme/_navbar.scss]: Ok I've tested, I get it. Ok it's due to logo itself wich has text next to illustration so, we are no choice even if I prefer `    background-position-y: -4px;` but let's open another PR :+1:

**Pull request merged**